src/Native/jitinterface/dllexport.h

```diff
@@ -16,25 +16,12 @@
 // ***
 // Define default call conventions
 // ***
-#ifndef _X86_
-
-#define DEFAULT_CALL_CONV
-#define __cdecl
-#define __stdcall
-
-#else // _X86_
-
-#ifndef __stdcall
-#define __stdcall          __attribute__((stdcall))
-#endif
-
+#if _X86_
 #ifdef PLATFORM_UNIX
-#define DEFAULT_CALL_CONV
+#define STDMETHODCALLTYPE  __attribute__((stdcall))
```

Comment:
STDMETHODCALLTYPE is `__cdecl` on Unix

Reply:
Ah, good catch. This header is really confusing. I don't understand why we renamed `__stdcall` to `__attribute__((stdcall))` if this is only supposed to be used on Windows (the renaming was the reason for the x86 Windows build break).